scripts/ci/kubernetes/kube/configmaps.yaml

```diff
@@ -27,7 +27,7 @@ data:
     logging_level = INFO
     executor = KubernetesExecutor
     parallelism = 32
-    load_examples = True
+    load_examples = False
```

Comment:
Load examples leads to a false positive because if the kubernetes volume doesn't contain DAGs (it's the case of running airflow with `git-sync` #3770) the DAGs exist and the test pass but it shouldn't.
This issue was present even before switching to the dockerised CI.